core/src/main/scala/io/finch/Encode.scala

```diff
@@ -23,6 +23,7 @@ trait LowPriorityEncodeInstances {
 
   type Json[A] = Aux[A, Application.Json]
   type Text[A] = Aux[A, Text.Plain]
+  type EventStream[A] = Aux[A, Text.EventStream]
```

Comment:
I'm not 100% sure we actually need these aliases here. I agree they are convenient, but given that users won't need to define instances by their own, I think, we can totally omit them here. What do you think?